luna/featurevis/transformations.py

```diff
@@ -73,33 +164,81 @@ def crop_or_pad(image, trans):
     if trans:
         trans_size = random.randint(-6, 6)
         image = tf.image.resize_with_crop_or_pad(
-            image, image.shape[1]-trans_size, image.shape[2]-trans_size)
+            image, image.shape[1] - trans_size, image.shape[2] - trans_size
+        )
     return image
 
 
-def vert_rotation(img, rotation):
-    """Randomly rotates the image.
+def vert_rotation(img, add_rotation):
```

Comment:
again, probably don't need the second argument.
Also, can we call this `ninety_degree_rotation` or something like that instead? I would not know what "rotating vertically" would mean.

Reply:
Resolved by giving transformations as function.